docs/dev/README.md

```diff
@@ -246,6 +246,12 @@ added to the profile via NVTX ranges.  See
 [the NVTX profiling guide](nvtx_profiling.md) for additional information on
 setting up the build for profiling and adding NVTX ranges.
 
+> Note: The Nsight Systems GUI is installed as part of the CUDA Toolkit, but is
```

Comment:
Curious, why is this using a quoted block?

Reply:
I guess it's just a side note in case someone wants to use the bundled version instead of downloading the standalone one.

Reply:
That's what I thought is covered by the "Note:" part.  Having it rendered as a quoted block makes it look like this was ripped from some other document which is a bit weird.  We're not quoting anything here.

Reply:
Ah I see what you mean. I think I was after the visual effect so it's less intrusive. I'll remove the quoted block.

Reply:
Removed the quoted block and reworded it a bit. PTAL